frontend/src/components/Library/Chart.tsx

```diff
@@ -236,9 +240,18 @@ const Chart = ({
           onChange={updateChartType}
           style={{ backgroundColor: "rgb(29, 36, 50)" }} // firefox's select element doesn't understand rgba...
         >
-          <option value="bar">Bar</option>
-          <option value="line">Line</option>
-          <option value="doughnut">Doughnut</option>
+          <option value="bar" disabled={isScatter}>
+            Bar
+          </option>
+          <option value="line" disabled={isScatter}>
+            Line
+          </option>
+          <option value="doughnut" disabled={isScatter}>
+            Doughnut
+          </option>
+          <option value="scatter" disabled={!isScatter}>
+            Scatter
+          </option>
```

Comment:
How about we don't show the scatter option if !isScatter, and we don't show the other ones if isScatter? Either that or modifying the colors so that it looks disabled because at the moment it doesnt look disabled
![image](https://github.com/user-attachments/assets/8460f8da-34cd-439c-8e8f-85dc82624746)

Reply:
Ughhhhhhhh - on my browser it looks very disabled! Are you on Firefox?

Reply:
Yes hahaha

Reply:
@RamiAwar I pushed a change to remove the select dropdown if isScatter. now looks like this:
![image](https://github.com/user-attachments/assets/86d67f5a-14f9-4238-ba9f-8df5db263933)